admin/schemas.php

```diff
@@ -196,7 +196,7 @@ protected function html_json(Schema $schema) {
         $form->addFieldsetClose();
 
         $form->addFieldsetOpen($this->getLang('import'));
-        $form->addElement(new \dokuwiki\Form\InputElement('file', 'schemafile'));
+        $form->addElement((new \dokuwiki\Form\InputElement('file', 'schemafile'))->attr('accept', '.json'));
```

Comment:
does adding method calls on (new foo()) work in 5.6? I remember it didn't work at one time but I forgot when it was introduced. Doesn't addElement() return the new element anyway, so attr could be called on the return of addElement() instead of (new *)?

Reply:
`(new foo())` works in 5.6, but you are right that it is unnecessary. Not sure why I implemented it that way 🤷.
Fixed.